spec/pagseguro/subscription_plan_spec.rb

```diff
@@ -92,8 +92,16 @@
           allow(subject).to receive(:code).and_return('12345')
         end
 
-        it 'should return correct url with code' do
-          expect(subject.url).to eq 'https://pagseguro.uol.com.br/v2/pre-approvals/request.html?code=12345'
+        context 'should return correct url with code' do
+          it 'to production' do
+            PagSeguro.configure { |config| config.environment = :production }
```

Comment:
But it shows a warning:

```
%  rspec spec/pagseguro/subscription_plan_spec.rb
........................[DEPRECATION] `environment=` is deprecated and will be removed. Please use configuration block instead.
.[DEPRECATION] `environment=` is deprecated and will be removed. Please use configuration block instead.
..

Finished in 0.34722 seconds (files took 1.86 seconds to load)
27 examples, 0 failures
```

You remember that task to remove warnings on specs, you did it :kissing: